app/controllers/api/v1/subscriptions_controller.rb

```diff
@@ -1,30 +1,30 @@
+# frozen_string_literal: true
+
 class Api::V1::SubscriptionsController < ApplicationController
   before_action :authenticate_user!
   before_action :user_is_subscriber?
   before_action :check_stripe_token
-  
+
   def create
-    begin
-      customer_id = get_customer(params[:stripeToken])
-
-      subscription = Stripe::Subscription.create({ 
-        customer: customer_id, 
-        plan: 'bds_subscription_plan' 
-      })
-
-      test_env?(customer_id, subscription)
-
-      status = Stripe::Invoice.retrieve(subscription.latest_invoice).paid
-
-      if status
-        current_user.update_attribute(:role, 'subscriber')
-        render json: {message: 'Transaction was successful'}
-      else
-        render_stripe_error('Transaction was unsuccessful')
-      end
-    rescue => error
-      render_stripe_error(error.message)
-    end 
+    customer_id = get_customer(params[:stripeToken])
+
+    subscription = Stripe::Subscription.create({
+                                                 customer: customer_id,
+                                                 plan: 'bds_subscription_plan'
+                                               })
```

Comment:
what happened here with formatting?

Reply:
Thats the formatting the ruby formatter gave